101-SQL-Injection-Attack-Prevention/nested/microsoft.sql/servers.v12.0.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "sqlServerName": {
+        "type": "string"
+      },
+      "location": {
+        "type": "string"
+      },
+      "administratorLogin": {
+        "type": "string",
+        "defaultValue": "sqladmin"
+      },
+      "administratorLoginPassword": {
+        "type": "securestring"
+      },
+      "tags": {
+        "type": "object"
+      }
+    },
+    "resources": [
+      {
+        "name": "[parameters('sqlServerName')]",
+        "type": "Microsoft.Sql/servers",
+        "apiVersion": "2015-05-01-preview",
```

Comment:
update apiVersion

Reply:
found this one as the latest in the resource explorer
![image](https://user-images.githubusercontent.com/24425344/42619853-d4cdda3c-85d6-11e8-8b30-9feebef39d5d.png)